spec/factories/users.rb

```diff
@@ -0,0 +1,13 @@
+FactoryBot.define do
+  factory :user do
+    nickname {Faker::Name.initials(number:3)}
+    email {Faker::Internet.free_email}
+    password {Faker::Internet.password(min_length:6)}
```

Comment:
```suggestion
    password {Faker::Internet.password(min_length:6)}password {Faker::Internet.password(min_length:6)}
    
```
パスワードが確実に英数字混合になるように記述を改めましょう！
現状だと英字のみや数字のみでパスワードが生成される可能性があります。